examples/stable/test/property/ic_api/in_replicated_execution/test/tests.ts

```diff
@@ -0,0 +1,134 @@
+import { expect, getCanisterActor, it, Test } from 'azle/test';
+import { execSync } from 'child_process';
+
+import { _SERVICE as Actor } from './dfx_generated/canister/canister.did';
+
+export function getTests(): Test {
+    return () => {
+        it('verifies init is running in replicated execution', async () => {
+            const actor = await getCanisterActor<Actor>('canister');
+
+            await checkInitIsInReplicatedExecution(actor, true);
+            await checkUpgradeIsInReplicatedExecution(actor, true);
+        });
+
+        it('verifies inspectMessage is running in replicated execution', async () => {
+            const actor = await getCanisterActor<Actor>('canister');
+
+            for (let i = 0; i < 10; i++) {
+                expect(
+                    await actor.getInspectMessageIsInReplicatedExecution()
+                ).toBe(true);
+            }
+        });
+
+        it('verifies non-replicated queries are not running in replicated execution', async () => {
+            const actor = await getCanisterActor<Actor>('canister');
+
+            for (let i = 0; i < 10; i++) {
+                expect(await actor.getQueryIsInReplicatedExecution()).toBe(
+                    false
+                );
+            }
+        });
+
+        it('verifies replicated queries are running in replicated execution', async () => {
+            const actor = await getCanisterActor<Actor>('canister');
+
+            for (let i = 0; i < 10; i++) {
+                expect(
+                    await actor.getQueryInReplicatedModeIsInReplicatedExecution()
+                ).toBe(true);
+            }
+        });
+
+        it('verifies composite queries are not running in replicated execution', async () => {
+            const actor = await getCanisterActor<Actor>('canister');
+
+            for (let i = 0; i < 10; i++) {
+                expect(
+                    await actor.getCompositeQueryIsInReplicatedExecution()
+                ).toBe(false);
+            }
+        });
+
+        it('verifies update is running in replicated execution', async () => {
+            const actor = await getCanisterActor<Actor>('canister');
+
+            for (let i = 0; i < 10; i++) {
+                expect(await actor.getUpdateIsInReplicatedExecution()).toBe(
+                    true
+                );
+            }
+        });
+
+        it('verifies heartbeat is running in replicated execution', async () => {
+            const actor = await getCanisterActor<Actor>('canister');
+
+            expect(
+                await actor.getHeartbeatIsInReplicatedExecution()
+            ).toStrictEqual([true]);
```

Comment:
Why does this use `toStrictEqual([true])` when other areas in this file do not? I think we should unify

Reply:
sure